README.md

```diff
@@ -105,6 +105,15 @@ The configuration for queues/topics can be changed or reset to default on PDA De
 | Broker URL  |       `localhost:9092`         |
 | Group ID    |       `inventory-operation`    |
 
+**Configuration details for external RabbitMQ server (default)**
+
+| Option   | Value       |
+|----------|-------------|
+| host     | `localhost` |
+| port     | `5672`      |
+| username | `guest`     |
+| password | `guest`     |
+
```

Comment:
We may need to include properties added in [PDA-1404](https://jira.parasoft.com/browse/PDA-1404). 
Please wait for the feedback.